xclim/indices/_conversion.py

```diff
@@ -695,3 +697,143 @@ def wind_chill_index(
 
     W.attrs["units"] = "degC"
     return W
+
+
+@declare_units(tasmin="[temperature]", tasmax="[temperature]")
+def potential_evapotranspiration(
+    tasmin: xr.DataArray, tasmax: xr.DataArray, tas: xr.DataArray, method: str = "BR65"
+) -> xr.DataArray:
+    """Potential evapotranspiration.
+
+    The daily or monthly potential for water evaporation from soil and transpiration by plants if the water supply is
+    sufficient, according to a given method.
+
+    Parameters
+    ----------
+    tasmin : xarray.DataArray
+      Minimum daily temperature.
+    tasmax : xarray.DataArray
+      Maximum daily temperature.
+    tas : xarray.DataArray
+      Mean daily temperature.
+    method : {"baierrobertson65", "hargreaves85", "thornthwaite48"}
+      Which method to use, see notes.
+
+    Returns
+    -------
+    xarray.DataArray
+        Potential evapotranspiration (daily or monthly).
+
+    Notes
+    -----
+    Available methods are:
+
+    - "baierrobertson65" or "BR65", based on [baierrobertson65]_.
+    - "hargreaves85" or "HG85", based on [hargreaves85]_.
+    - "thornthwaite48" or "TW48", based on [thornthwaite48]_.
+
+    References
+    ----------
+    .. [baierrobertson65] Baier, W., & Robertson, G. W. (1965). Estimation of latent evaporation from simple weather observations. Canadian journal of plant science, 45(3), 276-284.
+    .. [hargreaves85] Hargreaves, G. H., & Samani, Z. A. (1985). Reference crop evapotranspiration from temperature. Applied engineering in agriculture, 1(2), 96-99.
+    .. [thornthwaite48] Thornthwaite, C. W. (1948). An approach toward a rational classification of climate. Geographical review, 38(1), 55-94.
+    """
+
+    if method in ["baierrobertson65", "BR65"]:
+        tasmin = convert_units_to(tasmin, "degF")
+        tasmax = convert_units_to(tasmax, "degF")
+
+        latr = (tasmin.lat * np.pi) / 180
+        gsc = 0.082  # MJ/m2/min
+
+        # julian day fraction
+        jd_frac = (datetime_to_decimal_year(tasmin.time) % 1) * 2 * np.pi
+
+        ds = 0.409 * np.sin(jd_frac - 1.39)
+        dr = 1 + 0.033 * np.cos(jd_frac)
+        omega = np.arccos(-np.tan(latr) * np.tan(ds))
+        re = (
+            (24 * 60 / np.pi)
+            * gsc
+            * dr
+            * (
+                omega * np.sin(latr) * np.sin(ds)
+                + np.cos(latr) * np.cos(ds) * np.sin(omega)
+            )
+        )  # MJ/m2/day
+        re = re / 4.1864e-2  # cal/cm2/day
+
+        # Baier et Robertson(1965) formula
+        out = 0.094 * (
+            -87.03 + 0.928 * tasmax + 0.933 * (tasmax - tasmin) + 0.0486 * re
+        )
+        out = out.clip(0)
+        out.attrs["units"] = "mm/day"
```

Comment:
If we hard code the output units, we need to ensure the input frequency is daily (with `xr.infer_freq`) or we use `xclim.core.units.infer_sampling_units` to infer it.